src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -215,17 +216,18 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                     var fixes = fixCollection.Fixes;
                     var fixCount = fixes.Length;
 
+                    Func<CodeAction, SuggestedActionSet> getFixAllSuggestedActionSet = codeAction =>
+                                CodeFixSuggestedAction.GetFixAllSuggestedActionSet(codeAction, fixCount, fixCollection.FixAllContext,
+                                    workspace, _subjectBuffer, _owner._editHandler);
+
```

Comment:
That makes sense - however, couldn't we pass down the rest of the state to SuppressionSuggestedAction (some of which is already being passed) and just call CodeFixSuggestedAction.GetFixAllSuggestedActionSet() with this state? 

I prefer explicitly passing the arguments down as opposed to wrapping this state in a lambda. The lambda makes reading the code harder since I have to now jump between files to understand what the lambda is really doing.